columnflow/production/normalization.py

```diff
@@ -102,19 +102,20 @@ def normalization_weights(self: Producer, events: ak.Array, **kwargs) -> ak.Arra
     datasets with a leaf process contained in the leaf processes of the py:attr:`dataset_inst`.
     For stitching, the process_id needs to be reconstructed for each leaf process on a per event basis.
     """
-    # get the lumi
+    # get the lumi and read the process id column
     lumi = self.config_inst.x.luminosity.nominal
-
-    # read the cross section per process from the lookup table
     process_id = np.asarray(events.process_id)
 
+    # ensure all ids were assigned a cross section
     unique_process_ids = set(process_id)
-    invalid_ids = unique_process_ids - set(self.xs_table.rows[0])
+    invalid_ids = unique_process_ids - self.xs_process_ids
```

Comment:
this attribute does not exist

Reply:
Right, my bad, I forgot to stage this hunk. It's added now.